### PR TITLE
Fixing squid:S1192 -   String literals should not be duplicated

### PR DESCRIPTION
**Pull request by misgersameer**
This pull request is focused on resolving occurrences of Sonar rule "squid:S1192 -   String literals should not be duplicated" information about the issues here:
https://dev.eclipse.org/sonar/rules/show/squid:S1192

Please let me know if you have any questions.
Sameer Misger

### CODE REVIEW

File: src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryToken.java [hunk 1]
```diff
@@ -48,6 +48,7 @@
     private final String email;
     private final String token;
 
+    private static final String charecterSet = "UTF-8";
```
Comment (oleg-nenashev) [inline src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryToken.java]: no space after the line.
The variable should be also upper-case according to the Oracle coding style

File: src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpointTest.java [hunk 2]
```diff
@@ -34,22 +34,25 @@
  */
 public class DockerRegistryEndpointTest {
 
+	private static final String indexUrl = "https://index.docker.io/v1/";
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpointTest.java]: Wrong change. We intentionally keep constants in tests for the better readability

File: src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryToken.java (hunk 1 above)
Comment (carlossg) [inline src/main/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryToken.java]: in java 7 it'd be better to use StandardCharsets.UTF_8

File: src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpointTest.java (hunk 2 above)
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerRegistryEndpointTest.java]: Uhm, reconsidered it, LGTM

File: src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpointTest.java [hunk 3]
```diff
@@ -50,6 +50,7 @@
  */
 public class DockerServerEndpointTest {
     
+	private static final String missing = "missing";
```
Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpointTest.java]: Wrong tabulation

Comment (oleg-nenashev) [inline src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerEndpointTest.java]: Should be also uppercase name then

**Comment by oleg-nenashev** [comment]
ping. Any plans to address comments?

**Comment by oleg-nenashev** [comment]
Closing since there is no response from the submitter

**Pull request closed without merging**